src/Test/UnitTests/unit_tests.jl

```diff
@@ -58,8 +58,12 @@ function test_model_solution(model, config;
     )
     config.solve || return
     atol, rtol = config.atol, config.rtol
+    @test MOI.get(model, MOI.SolverName()) isa AbstractString
```

Comment:
I would suggest creating separate tests for each attribute instead of adding additional complexity to this test.